crates/qt-build-utils/src/lib.rs

```diff
@@ -450,7 +450,7 @@ impl QtBuild {
     /// Run moc on a C++ header file and save the output into [cargo's OUT_DIR](https://doc.rust-lang.org/cargo/reference/environment-variables.html).
     /// The return value contains the path to the generated C++ file, which can then be passed to [cc::Build::files](https://docs.rs/cc/latest/cc/struct.Build.html#method.file),
     /// as well as the path to the generated metatypes.json file, which can be passed to [register_qml_types](Self::register_qml_types).
-    pub fn moc(&mut self, input_file: impl AsRef<Path>) -> MocProducts {
+    pub fn moc(&mut self, input_file: impl AsRef<Path>, uri_opt: &str) -> MocProducts {
```

Comment:
Could you add some documentation about what `uri_opt` does to the doc lines above?

Also assume `uri_opt` should always be mandatory and not a `Option<&str>` ? :thinking:

Reply:
I added some documentation. I did not know how it should be documented so I chose something that looks good.